### PR TITLE
Create arduino-bme280

### PR DESCRIPTION
**Pull request by jacobrussell**
Submitting pull request to correct repo.

### CODE REVIEW

File: configs/adafruit/arduino-bme280 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Adafruit BME280 Library",
```
Comment (ivankravets) [inline configs/adafruit/arduino-bme280]: I propose to change name of library to `Adafruit_BME280`. We have practice to use main `include` for the library name.

The user knows to use this lib is enough to `#include <Adafruit_BME280.h>`

Thanks.

File: configs/adafruit/arduino-bme280 [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "keywords": "sensor, temp, humidity, pressure",
+  "description": "Arduino library for BME280 humidity and pressure sensors",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/adafruit/Adafruit_BME280_Library.git"
+  },
+  "frameworks": "arduino",
+  "platforms": "atmelavr"
+}
```
Comment (ivankravets) [inline configs/adafruit/arduino-bme280]: The last thing, please add `.json` extension to this file. It should be `arduino-bme280.json`.

**Comment by jacobrussell** [comment]
Thanks, sorry for the problems, new to the ecosystem.

**Comment by ivankravets** [comment]
Done! http://platformio.org/#!/lib/show/166/Adafruit_BME280

Thank you too! No problems. We are glad to merge new configs because don't have time to add them :( if you see that some popular library is not in our registry, fill free to add it.

**Pull request merged**